### PR TITLE
initial testing harness

### PR DESCRIPTION
**Pull request by ericmcbride**
This covers issue https://github.com/MikeyDunn/S.T.A.L.K.E.R./issues/2
This will also open up a new issue https://github.com/MikeyDunn/S.T.A.L.K.E.R./issues/5 since there wasn't that many details in the original issue.

### CODE REVIEW

**Comment by levi-rs** [comment]
LGTM :family:

File: tests/test_reddit.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import unittest
+from unittest import mock
+
+import pytest
+import requests
+from requests.exceptions import HTTPError
+
+from src.social.reddit import Reddit
+
+#TODO: Move _mock_response to a conf.py or something similiar, add more tests
```
Comment (prestontimmons) [inline tests/test_reddit.py]: ur a TODO

Comment (ericmcbride) [reply tests/test_reddit.py]: Dayum rekt

File: tests/test_reddit.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import json
+import time
+import unittest
+from unittest import mock
+
+import pytest
+import requests
+from requests.exceptions import HTTPError
+
+from src.social.reddit import Reddit
```
Comment (MikeyDunn) [inline tests/test_reddit.py]: seeing this on pytest

```
=================================== ERRORS ====================================
____________________ ERROR collecting tests/test_reddit.py ____________________
ImportError while importing test module 'D:\Dev\S.T.A.L.K.E.R\tests\test_reddit.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests\test_reddit.py:10: in <module>
    from src.social.reddit import Reddit
E   ModuleNotFoundError: No module named 'src'
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!
=========================== 1 error in 0.50 seconds ===========================```

**Comment by MikeyDunn** [comment]
LGTM. Thanks @ericmcbride

**Pull request merged**